src/services/configInjection.ts

```diff
@@ -0,0 +1,109 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+import { dbLogger } from '../logger';
+import Mapping from '../domain/Mapping';
+import { expandEnvVarsInMap } from '../util/env';
+import Connection from '../Connection';
+import SyncConfig from '../domain/SyncConfig';
+import KeyValuePair from '../domain/KeyValuePair';
+import { INJECTED_CONFIG_TABLE } from '../constants';
+
+/**
+ * Reads and returns the package.json contents.
+ *
+ * @returns {*}
+ */
+export function getPackageMetadata(): any {
+  return JSON.parse(fs.readFileSync(path.join(__dirname, '../../package.json')).toString());
+}
+
+/**
+ * Gets all the default config / environment variables
+ * that are always available in the injected config table.
+ *
+ * @returns {Mapping<string>}
+ */
+function getDefaultSystemVars(): Mapping<string> {
```

Comment:
I don't think the doc block and the function are trying to do the same thing?

Reply:
Actually, it is but the thing is we only have a single param to be passed as a default. It is for now a placeholder for other potential default configs that are to be injected later.

Reply:
Specially `gets all the environment variables`?

Reply:
All I see is the version.

Reply:
For now I've removed everything and have put an empty function as a placeholder for later defaults to inject.